distribution/src/main/resources/bin/elasticsearch

```diff
@@ -74,6 +81,17 @@ ES_HOME=`dirname "$SCRIPT"`/..
 # make ELASTICSEARCH_HOME absolute
 ES_HOME=`cd "$ES_HOME"; pwd`
 
+if [ -z "$ES_JVM_OPTIONS" ]; then
+    for jvm_options in "$ES_HOME"/config/jvm.options \
+                      /etc/elasticsearch/jvm.options; do
+        if [ -r "$jvm_options" ]; then
+            ES_JVM_OPTIONS=$jvm_options
+            break
+        fi
+    done
+fi
+
+ES_JAVA_OPTS="$(parse_jvm_options "$ES_JVM_OPTIONS") $ES_JAVA_OPTS"
```

Comment:
@dakrone I've gone back and forth on this a few times in my mind. Some JVM conflicting options will cause the JVM to not start, and some are last-write wins. I don't think that people should be relying on this *ever*, and I so I don't think that it should be documented. Do you still think it should be documented?